src/styles/calendar.scss

```diff
@@ -0,0 +1,36 @@
+.calendar {
+  position: relative;
+  width: 800px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  margin: 0 auto;
+  padding-left: 47px;
+
+  &__day {
+    background-color: #eee;
+    width: $cell-size;
+    height: $cell-size;
```

Comment:
Variables `$cell-size` should be defined in a separate file (e.g., `_variables.scss`) to improve maintainability and reusability across different stylesheets.